app/steps/ui/thankyou/index.js

```diff
@@ -10,6 +10,7 @@ class ThankYou extends Step {
 
     getContextData(req) {
         const ctx = super.getContextData(req);
+        ctx.ccdReferenceNumber = '';
```

Comment:
Can we have a unit test to check that `ctx.ccdReferenceNumber` is set to an empty string if the case id doesn't exist?